code/dqe/xmarketalert.q

```diff
@@ -0,0 +1,7 @@
+\d .dqe 
+xmarketalert:{[tab]                                     // alerts user when bid has exceeded the ask in market data
+  data:select from tab where bid>ask;
```

Comment:
should we have a check here that these columns exist? or are we relying on error catching at a higher level in DQE?

e.g. what happens if a user passes `trade` table as argument to this function instead of `quote`?

Reply:
@tszwailam can you answer this question?

Reply:
@jonathonmcmurray Sorry I answered in the bottom and forgot to answer here - 
Error handling takes care of this and it doesn't require a conditional here